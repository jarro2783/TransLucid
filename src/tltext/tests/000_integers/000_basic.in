%%
0;;
1;;
10;;
100;;
~1;;
~100;;

0 + 0;;
0 + 10;;
10 + 0;;
~100 + 0;;
0 + ~100;;

10 - 0;;
0 - 10;;
10 - 10;;
100 - 200;;
100 - ~100;;

10 * 0;;
10 * 1;;
10 * 10;;
4 * 5;;
~10 * 5;;
~10 * ~20;;

10 / 1;;
10 / 2;;
10 / 4;;
~10 / 2;;
10 / ~2;;
~10 / ~2;;

23 % 4;;

0 < 1;;
1 < 1;;
2 < 1;;

0 <= 1;;
1 <= 1;;
2 <= 1;;

0 > 1;;
1 > 1;;
2 > 1;;

0 >= 1;;
1 >= 1;;
2 >= 1;;

0 == 1;;
1 == 1;;
2 == 1;;

0 != 1;;
1 != 1;;
2 != 1;;
