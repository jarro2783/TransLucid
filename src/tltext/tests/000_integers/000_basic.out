0
1
10
100
~1
~100
0
10
10
~100
~100
10
~10
0
~100
200
0
10
100
20
~50
200
10
5
2
~5
~5
5
3
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
